### PR TITLE
Add missing German translations part 2

### PR DESCRIPTION
**Pull request by stefangrotz**
Follow up for #3005
Missing translations + this correction:

```
"initial_prompt_review": "Initial-Prompts geprüft",
-> "Initiale Prompt-Prüfung",
```

These strings are still untranslated, I plan to translate them in a future PR:

* labelling.json (56!)
* leaderboard.json (21)
* tasks.json (15)

### CODE REVIEW

File: website/public/locales/de/common.json [hunk 1]
```diff
@@ -22,6 +23,8 @@
   "faq": "FAQ",
   "guidelines": "Richtlinien",
   "github": "GitHub",
+  "hide": "Hide",
+  "hugging_face": "HuggingFace",
```
Comment (notmd) [inline website/public/locales/de/common.json:27]: I dont know why we translate this lol

**Review by notmd — APPROVED** [review]
Thanks

**Comment by notmd** [comment]
Opps pre commit failed

File: website/public/locales/de/common.json (hunk 1 above)
Comment (stefangrotz) [reply website/public/locales/de/common.json:27]: Just part of the general structure I guess. But I forgot to translate hide and copy here, ups.

**Comment by stefangrotz** [comment]
Should be fine now.

**Pull request merged**